internal/ethapi/api.go

```diff
@@ -1989,6 +1993,24 @@ func (s *TransactionAPI) GetRawTransactionByHash(ctx context.Context, hash commo
 func (s *TransactionAPI) GetTransactionReceipt(ctx context.Context, hash common.Hash) (map[string]interface{}, error) {
 	tx, blockHash, blockNumber, index, err := s.b.GetTransaction(ctx, hash)
 	if tx == nil || err != nil {
+		block, err := s.b.BlockByHash(ctx, hash)
```

Comment:
In celo we added an extra rpc (`eth_getBlockReceipt`) I think we should maintain this approach as opposed to overloading `eth_getTransactionReceipt`?